CHANGES.md

```diff
@@ -75,6 +75,10 @@ OpenSSL 3.3
 
     *Neil Horman*
 
+ * Added support for requesting CRL in CMP.
+
+    *Rajeev Ranjan, Siemens AG*
```

Comment:
We do not add affiliation to CHANGES.md entries in this way.

Reply:
However we have text like: `This work was sponsored by ...` in some entries below, so I think that would be acceptable?